src/widget/form/settings/generalform.cpp

```diff
@@ -515,7 +639,7 @@ void GeneralForm::onThemeColorChanged(int)
 bool GeneralForm::eventFilter(QObject *o, QEvent *e)
 {
     if ((e->type() == QEvent::Wheel) &&
-         (qobject_cast<QComboBox*>(o) || qobject_cast<QAbstractSpinBox*>(o) || qobject_cast<QCheckBox*>(o)))
+         (qobject_cast<QComboBox*>(o) || qobject_cast<QAbstractSpinBox*>(o) || qobject_cast<QTabBar*>(o) || qobject_cast<QCheckBox*>(o)))
```

Comment:
This whole eventFilter thing looks crap to me. Anyway, this must be handled in a separate topic to relly solve it. So, just leave it as is for now.